src/ReverseProxy/Management/ProxyConfigManager.cs

```diff
@@ -200,17 +231,75 @@ private async Task ReloadConfigAsync()
             Log.ErrorApplyingConfig(_logger, ex);
         }
 
-        if (newConfig.ChangeToken.ActiveChangeCallbacks)
+        ListenForConfigChanges();
+    }
+
+    private static void ValidateConfigProperties(IProxyConfig config)
+    {
+        if (config == null)
```

Comment:
In order to follow https://github.com/dotnet/aspnetcore/issues/39323?
```suggestion
        if (config is null)
```

Reply:
Fair, but that's not a style we've adopted yet in this repo and I don't want to be inconsistent. Can you file an issue for recommended style changes?